README.md

```diff
@@ -175,7 +175,4 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
-A copy of the license is available in the repository's [LICENSE](./license.txt) file.
-
-[](Esri Tags: ArcGIS Web Mapping Leaflet AppBuilder Hackerlabs Labs JavaScript)
-[](Esri Language: JavaScript)
```

Comment:
because GitHub's latest markdown parser no longer hides them, we can (and should).

https://github.com/esri/geodev-hackerlabs#licensing